libs/gltfio/src/AssetLoader.cpp

```diff
@@ -46,6 +46,9 @@
 #define CGLTF_IMPLEMENTATION
 #include <cgltf.h>
 
+#define STB_IMAGE_IMPLEMENTATION
+#include <stb_image.h>
```

Comment:
`AssetLoader` doesn't use STB, can we fix this another way, perhaps by adding the define to `ResourceLoader` instead?

Reply:
Ah, yes, that does make more since 👍